### PR TITLE
feat: update footer links

### PR DESCRIPTION
**Pull request by luzzif**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,3 +25,30 @@ Typescript.
...
+## Getting Started
+
+Start by installing the dependencies with `pnpm`:
+
+```
+pnpm i
+```
+
+An env variable is required to start the landing page and it determines if the
+landing page should run in dev mode or not. Create a `.env.local` file at the
```
Comment (guerrap) [inline README.md]: Why do we talk about the landing page here?

File: commons/constants.ts [hunk 2]
```diff
@@ -16,65 +19,53 @@ export const FOOTER_LINKS: FooterLink[] = [
...
+            // TODO: add link to faq once we have them
+            // {
+            //     Component: Link,
+            //     title: "Faq",
+            //     to: "/faq",
+            // },
             {
-                title: "FAQ",
-                to: "/faq",
+                title: "Documentation",
```
Comment (guerrap) [inline commons/constants.ts]: Why do we need a link to the docs on the docs site?

File: README.md (hunk 1 above)
Comment (luzzif) [reply README.md]: Whoops, it's because of copy/paste. Addressed in 46a4df7af6ee4f7f521c3abacb14e5a17dc89209

File: commons/constants.ts (hunk 2 above)
Comment (luzzif) [reply commons/constants.ts]: Again, landing page copy-paste. Replaced with a link to the app in 930b930642777ebfb6efe73832c03acd899c4f43

**Review by guerrap — APPROVED** [review]

**Pull request merged**